Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```diff
@@ -0,0 +1,300 @@
+﻿- type: entity
+  name: slime
```

Comment:
i could be wrong but if this is never spawned directly add `abstract: true`

Reply:
This is left over from the beginning of the work, I'll translate it into a good blue slime